README.md

```diff
@@ -22,6 +22,7 @@
 - [Flagship](https://flagship.io) - Feature management and feature experimentation platform.
 - [Flagsmith](https://flagsmith.com) - An all-in-one platform for developing, implementing, and managing your feature flags.
 - [Growthbook](https://growthbook.io) - An open source platform to help companies make data-driven product decisions with feature flags and A/B tests.
+- [GO Feature Flag](https://gofeatureflag.org) - GO Feature Flag is a simple, complete and lightweight self-hosted feature flag solution 100% Open Source.
```

Comment:
What about just "A simple, lightweight self-hosted open source feature flag solution?

Reply:
This has been resolved in the most recent merge.